showcase/src/assets/examples/typography/mat.html

```diff
@@ -0,0 +1,6 @@
+<div class="mat-display-4">Display 4</div>
+<div class="mat-display-3">Display 3</div>
```

Comment:
Could you maybe change the text from "Display 4" to "Mat-Display-4" so it is clear what is the class being used?

The same for the others in this section